CHARTER.md

```diff
@@ -108,9 +108,13 @@ process as standard proposals.
 ## Code of Conduct Enforcement
 
 Removing or disciplining an ART member due to a violation of the [Arcalot Code of
-Conduct](https://github.com/arcalot/.github/blob/main/CODE_OF_CONDUCT.md) is performed
-by the chairs as a pull request to the ART. Only the the chairs vote on these matters,
-with an approval by 75% of the active chairs required for the matter to pass.
+Conduct](https://github.com/arcalot/.github/blob/main/CODE_OF_CONDUCT.md) is initiated
+by any chair as a pull request to the ART. Only the the chairs vote on these matters,
+with an approval by unanimous consent of the active chairs and with a 7 calendar day
+voting period. In the case of a dissenting vote, unanimous consent is not achieved, and
+the matter moves to the normal proposal process as described above. An appeal may be
```

Comment:
"the matter" is the removal (or "discipline" -- how does *that* work if it's not a removal?) of a member. I really think any time we throw out fluff words like "matter" we should instead use something more concrete and direct.

Reply:
I just don't see that it is a fluff word at all. The specificity of the word "matter" here is implied by the context of the paragraph. To me, the more explicit the language is, the more it sounds like legalease and the less able I am to absorb the meaning without re-reading it several times.